MuxExoPlayer/src/r2_14_1/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java

```diff
@@ -302,10 +313,15 @@ public void onUpstreamDiscarded(EventTime eventTime, MediaLoadData mediaLoadData
   }
 
   @Override
-  public void onVideoSizeChanged(AnalyticsListener.EventTime eventTime, int width, int height,
-      int unappliedRotationDegrees, float pixelWidthHeightRatio) {
-    sourceWidth = width;
-    sourceHeight = height;
+  public void onVideoSizeChanged(EventTime eventTime, VideoSize videoSize) {
```

Comment:
did ExoPlayer change the signature here? And why did we not need this before? Did this fix an issue that existed in 2.14.1? I think I lost track of where this specific change came from, but perhaps it was from a previous PR.